apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-button/index.ts

```diff
@@ -75,18 +80,6 @@ domReady( () => {
 					dispatch( 'automattic/launch' ).openSidebar();
 					delayedSavePost();
 					break;
-				case FOCUSED_LAUNCH_FLOW:
-					// Save post in the background while focused launch flow opens
-					dispatch( 'automattic/launch' ).openFocusedLaunch();
-					delayedSavePost();
-					break;
-				case SITE_LAUNCH_FLOW:
-					// Save post first before redirecting to launch url
-					( async () => {
-						await savePost();
-						window.top.location.href = launchUrl;
-					} )();
```

Comment:
See previous comment about scope creep.

Reply:
I'm fine with keeping this but it's just dead code since the button is not there in these 2 cases. 
Also, I think it may lead to confusion to have these around because of the workarounds we have in place for AnchorFM (see the `@TODO` above).

Reply:
Yeah, I agree that it's better to remove it.